### ISSUE TITLE
 How to retrain the model?

### ISSUE THREAD

**Issue by MarieMaria** [oarriaga/face_classification#60]
Hello,

I have been facing trouble when I try to retrain the train_emotion_classifier model. The model don't finish all 1000 but stop ones at epoch 00156 and the second time at epoch 00136.

Can you please explain me the reason? 
 Thanks for your help

**Comment by nrahmadinda** [comment]
hey im facing the same problem, do you have any solution?

**Comment by nrahmadinda** [comment]
hai, i am facing the same problem do yo have the solution?

**Comment by nrahmadinda** [comment]
hello, i solve the problem by comment line early_stop

**Comment by oarriaga** [comment]
The training script is not meant to use all the 1000 epochs but to stop using the EarlyStop keras callback whenever the loss is not decreasing any further.

**Comment by NomiMalik0207** [comment]
But every time its stops after 100 or 150 epochs...
is this okay ?

**Comment by MarieMaria** [comment]
hello,

Yes its ok. The model shouldn't finish training all 1000 epochs. The author
added an EarlyStop callback which stops the training whenever the loss is
not decreased after several epochs.
I hope this answers your question.

Best regards
Mariem


2018-05-24 7:16 GMT+02:00 NomiMalik0207 <notifications@github.com>:
